plugin/secrets_access_token.go

```diff
@@ -73,8 +74,9 @@ func (b *backend) secretAccessTokenResponse(ctx context.Context, s logical.Stora
 
 	return &logical.Response{
 		Data: map[string]interface{}{
-			"token":      token.AccessToken,
-			"expires_at": token.Expiry,
+			"token":           token.AccessToken,
+			"token_ttl":       time.Now().Sub(token.Expiry),
```

Comment:
Thanky. In case you're curious, here's the magic: https://github.com/hashicorp/vault/blob/d3017e259fc23897896e2ab8c59cda8c9bdc0ae3/command/format.go#L249-L255

Do we need to make both of those times UTC for safety? `time.Now().UTC().Sub(token.Expiry.UTC())`?